plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -0,0 +1,418 @@
+				'batch-size' => 500,
+				'start-from' => 0,
+			)
+		);
+
+		$batch_count    = 1;
+		$total_time     = 0;
+		$failed_ids     = array();
+		$processed      = 0;
+		$order_id_start = (int) $assoc_args['start-from'];
+		$order_count    = $this->get_verify_order_count( $order_id_start );
+		$batch_size     = ( (int) $assoc_args['batch-size'] ) === 0 ? 500 : (int) $assoc_args['batch-size'];
+
+		$progress = WP_CLI\Utils\make_progress_bar( 'Order Data Verification', $order_count / $batch_size );
+
+		if ( ! $order_count ) {
+			return WP_CLI::warning( __( 'There are no orders to verify, aborting.', 'woocommerce' ) );
+		}
+
+		while ( $order_count > 0 ) {
+			WP_CLI::debug(
+				sprintf(
+				/* Translators: %1$d is the batch number, %2$d is the batch size. */
+					__( 'Beginning verification for batch #%1$d (%2$d orders/batch).', 'woocommerce' ),
+					$batch_count,
+					$batch_size
+				)
+			);
+
+			$order_ids        = $wpdb->get_col(
+				$wpdb->prepare(
+					"SELECT ID FROM $wpdb->posts WHERE post_type = 'shop_order' AND ID > %d ORDER BY ID ASC LIMIT %d",
+					$order_id_start,
+					$batch_size
+				)
+			);
+			$batch_start_time = microtime( true );
+			$failed_ids       = $failed_ids + $this->post_to_cot_migrator->verify_migrated_orders( $order_ids );
+			$processed       += count( $order_ids );
+			$batch_total_time = microtime( true ) - $batch_start_time;
+			$batch_count ++;
+			$total_time += $batch_total_time;
+
+			$progress->tick();
+
+			WP_CLI::debug(
+				sprintf(
+				// Translators: %1$d is the batch number, %2$f is time taken to process batch.
+					__( 'Batch %1$d (%2$d orders) completed in %3$f seconds', 'woocommerce' ),
+					$batch_count,
+					count( $order_ids ),
+					$batch_total_time
+				)
+			);
+
+			$order_id_start  = max( $order_ids );
+			$remaining_count = $this->get_verify_order_count( $order_id_start, false );
+			if ( $remaining_count === $order_count ) {
+				return WP_CLI::error( __( 'Infinite loop detected, aborting. No errors found.', 'woocommerce' ) );
+			}
+			$order_count = $remaining_count;
+		}
+
+		$progress->finish();
+		WP_CLI::log( __( 'Verification completed.', 'woocommerce' ) );
+
+		if ( 0 === count( $failed_ids ) ) {
+			return WP_CLI::success(
+				sprintf(
+				/* Translators: %1$d is the number of migrated orders and %2$f is time taken */
+					_n( '%1$d order was verified, in %2$f seconds', '%1$d orders were verified in %2$f seconds', $processed, 'woocommerce' ),
+					$processed,
+					$total_time
+				)
+			);
+		} else {
+			$errors = print_r( $failed_ids, true );
+
+			return WP_CLI::error(
+				sprintf(
+				/* Translators: %1$d is the number of migrated orders, %2$f is time taken, %3$f is number of errors and$4%s is formatted array of order ids. */
+					_n( '%1$d order was verified, in %2$f seconds. %3$f error was found: %4$s', '%1$d orders were verified in %2$f seconds. %3$f errors were found %4$s', $processed, 'woocommerce' ),
+					$processed,
+					$total_time,
```

Comment:
It's missing `count( $failed_ids ),` over here.

Reply:
done in https://github.com/woocommerce/woocommerce/pull/32938/commits/387fd1e5cde3cf4950d161b6572ef1bf2349390e